sql/core/src/main/scala/org/apache/spark/sql/sources/ddl.scala

```diff
@@ -18,6 +18,7 @@
 package org.apache.spark.sql.sources
 
 import scala.language.implicitConversions
+import scala.language.existentials
```

Comment:
I know it's trivial, but this is not ordered correctly, even. This was a change I included in a PR I was working on this weekend, and just submitted: https://github.com/apache/spark/pull/4948/files